src/open_inwoner/openzaak/import_export.py

```diff
@@ -115,6 +117,74 @@ class CatalogusConfigImport:
     zaak_status_type_configs_imported: int = 0
     zaak_resultaat_type_configs_imported: int = 0
 
+    @staticmethod
+    def _get_url_root(url: str):
+        parsed = urlparse(url)
+        if not any((parsed.scheme, parsed.netloc)):
+            raise ValueError(f"{url} is not a valid URL")
+
+        return f"{parsed.scheme}://{parsed.netloc}"
+
+    @classmethod
+    def _lines_iter_from_jsonl_stream_or_string(cls, stream_or_string):
```

Comment:
```suggestion
    def _lines_iter_from_jsonl_stream_or_string(cls, stream_or_string: io.IOBase | str) -> Iterator[str]:
```
with `from typing import Iterator`. Or some other return type.